hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestRecordLevelIndex.scala

```diff
@@ -229,19 +229,14 @@ class TestRecordLevelIndex extends RecordLevelIndexTestBase {
   }
 
   @ParameterizedTest
-  @CsvSource(value = Array(
-    "COPY_ON_WRITE,6", "COPY_ON_WRITE,8", "MERGE_ON_READ,6", "MERGE_ON_READ,8"
-  ))
-  def testRLIWithDelete(tableType: String, tableVersion: Int): Unit = {
-    val hudiOpts = commonOpts ++ Map(
-      DataSourceWriteOptions.TABLE_TYPE.key -> tableType,
-      HoodieTableConfig.VERSION.key() -> tableVersion.toString,
-      HoodieWriteConfig.WRITE_TABLE_VERSION.key() -> tableVersion.toString)
+  @EnumSource(classOf[HoodieTableType])
```

Comment:
The full suite runs with v6 with `TestRecordLevelIndexTableVersionSix` so we don't need to override at this level as well. This lead to 4 repeated test cases.

Reply:
sg